src/components/common/FilterGroup/filters/FilterDateRange.tsx

```diff
@@ -2,11 +2,12 @@ import React from 'react';
 
 import { Grid, Theme } from '@mui/material';
 import { makeStyles } from '@mui/styles';
-import { format, isValid } from 'date-fns';
```

Comment:
format seems inappropriate since a Date without time does not need to be localized to timezones

Reply:
Nice catch.